src/Maestro/tests/Maestro.ScenarioTests/MaestroScenarioTestBase.cs

```diff
@@ -169,13 +233,61 @@ public async Task<AsyncDisposableValue<string>> CreateSubscriptionAsync(string s
                 return AsyncDisposableValue.Create(subscriptionId, async () =>
                 {
                     TestContext.WriteLine($"Cleaning up Test Subscription {subscriptionId}");
-                    await RunDarcAsync("delete-subscriptions", "--id", subscriptionId, "--quiet").ConfigureAwait(false);
+                    try
+                    {
+                        await RunDarcAsync("delete-subscriptions", "--id", subscriptionId, "--quiet").ConfigureAwait(false);
+                    }
+                    catch (MaestroTestException)
+                    {
+                        // If this throws an exception the most likely cause is that the subscription was deleted as part of the test case
+                    }
                 });
             }
 
             throw new MaestroTestException("Unable to create subscription.");
         }
 
+        public async Task<string> GetSubscriptionInfo(string subscriptionId)
+        {
+            return await RunDarcAsync("get-subscriptions", "--ids", subscriptionId).ConfigureAwait(false);
+        }
+
+        public async Task<string> GetSubscriptions(string channelName)
+        {
+            return await RunDarcAsync("get-subscriptions", "--channel", $"\"{channelName}\"");
+        }
+
+        public async Task ValidateSubscriptionInfo(string subscriptionId, string expectedSubscriptionInfo)
+        {
+            string subscriptionInfo = await GetSubscriptionInfo(subscriptionId);
+            StringAssert.AreEqualIgnoringCase(expectedSubscriptionInfo, subscriptionInfo);
+        }
+
+        public async Task SetSubscriptionStatus(bool enableSub, string subscriptionId = null, string channelName = null)
+        {
+            string actionToTake = enableSub ? "--enable" : "-d";
+
+            if (channelName != null)
+            {
+                await RunDarcAsync("subscription-status", actionToTake, "--channel", channelName, "--quiet").ConfigureAwait(false);
+            }
+            else
+            {
+                await RunDarcAsync("subscription-status", "--id", subscriptionId, actionToTake, "--quiet").ConfigureAwait(false);
+            }
+        }
+
+        public async Task<string> DeleteSubscriptionsForChannel(string channelName)
+        {
+            string[] parameters = { "delete-subscriptions", "--channel", $"\"{channelName}\"", "--quiet" };
+            return await RunDarcAsync("delete-subscriptions", "--channel", $"{channelName}", "--quiet");
```

Comment:
No need to interpolate a single string variable, just pass it

```suggestion
            return await RunDarcAsync("delete-subscriptions", "--channel", channelName, "--quiet");
```